src/windows/com.cc

```diff
@@ -15,16 +15,17 @@
  */
 
 #include "src/windows/com.h"
+#include "src/log.h"
 
 HRESULT drivelist::com::Initialize() {
   HRESULT result;
 
-  // Initialize COM
+  drivelist::Debug("Initializing COM");
```

Comment:
The COM stuff is just because we're still using WMI isn't it, and so it'll all go when we've finally got rid of all the code using WMI (by replacing it with lower-level APIs)?